src/rdbms-connect/README.rst

```diff
@@ -0,0 +1,37 @@
+Microsoft Azure CLI 'rdbms-connect' Extension
+==========================================
+
+This extension enables the command to connect to Azure Database for MySQL and Azure Database for PostgreSQL flexible server instances. .  
+
+-----
+Usage
+-----
+
+To install the extension separately can run:
+
+::
+    az postgres flexible-server connect -n testServer -u username -p password --postgres-query "select * from pg_user;" --output table
+
+::
+    az mysql flexible-server connect -n testServer -u username -p password --mysql-query "select host, user from mysql.user;" --output table
```

Comment:
This seems how to use, not how to install. It should be:
```
To install this extension using the below CLI command

> az extension add --name rdbms-connect

```

Reply:
Even with dynamically loading the extension?  Since this is the readme for the extension, it makes sense let me make that edit.